prov/verbs/src/verbs_info.c

```diff
@@ -327,18 +340,21 @@ int fi_ibv_check_tx_attr(const struct fi_tx_attr *attr,
 	if (attr->size > info->tx_attr->size) {
 		VERBS_INFO(FI_LOG_CORE,
 			   "Given tx_attr->size is greater than supported\n");
+		FI_INFO_CHECK(&fi_ibv_prov, (info->tx_attr), attr, size, FI_TYPE_SIZE_T);
```

Comment:
Do we need the parenthesis around info->tx_attr? If it's needed it should be moved into the macro.